CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+    public class Jester
+    {
+        public IJokeServiceInterface? JokeService { get; set; }
+        public IPrintJokeInterface? Printer { get; set; }
+
+        public Jester(IJokeServiceInterface? jokeService, IPrintJokeInterface? printer)
```

Comment:
```suggestion
        public Jester(IJokeServiceInterface jokeService, IPrintJokeInterface printer)
```
Should not be nullable and check that is enforced